cartesi-rollups/build-dapps/run-dapp.md

```diff
@@ -74,7 +74,20 @@ docker buildx bake --load
 
 ## Running
 
-To run the application, you can start an environment that includes a local blockchain with the Cartesi smart contracts deployed, as well as a Cartesi layer-2 node executing the DApp's back-end logic:
+
+To run the application, you can start an environment that includes a local blockchain with the Cartesi smart contracts deployed, as well as a Cartesi layer-2 node executing the DApp's back-end logic. Before this, check if your Docker supports `riscv` by running:
+
+```shell
+docker buildx ls
+```
+
+If you do not see `riscv` in the results, install this dependency by running:
+
+```shell
+apt install qemu-user-static
+```
+
+Now we can start the environment:
```

Comment:
Another thing is, when you say "If you do not see `riscv` in the results, install this dependency by runing:". You can specify what the user should see to avoid mistakes.

"If you do not see `linux/riscv64` in the platforms list, install qemu by runing:".

Reply:
If you feel the need, you can add an explanation on qemu so the user knows what he is installing.
Based on [qemu docs](https://www.qemu.org/docs/master/about/index.html):
"QEMU is a generic and open source machine emulator and virtualizer that will be used by Docker to emulate RISCV instructions to build a Cartesi Machine for your DApp. After installing qemu the platform `linux/riscv64` should appear in the platforms list."